esp/templates/errors/program/wronggrade.html

```diff
@@ -4,15 +4,16 @@
 
 {% block content %}
 
-<h1>{{ program.niceName }} Student Registration&mdash; Error</h1>
+<h1>{{prog.niceName }} Student Registration &mdash; Error</h1>
```

Comment:
I'd just remove the em dash entirely, to be honest!

Reply:
Sounds good